pyperf/_cpu_utils.py

```diff
@@ -2,11 +2,16 @@
 import os
 import re
 
-from pyperf._utils import sysfs_path, proc_path, read_first_line
+from pyperf._utils import sysfs_path, proc_path, read_first_line, FREE_THREADING
 
 try:
     # Optional dependency
-    import psutil
+    # Currently there is a packaging issue for PEP-703,
+    # Until then psutil is disabled as a workaround.
+    if FREE_THREADING:
```

Comment:
Mmmh. Maybe instead of copying the same comment in many files, you can add a constant and reuse it, to have a single comment where the constant is defined. I'm making this comment since I would like to suggest to copy the URL to the bug.

Reply:
Hmm, would you like to propose the suggestion more concretely?
I think that this PR will be reverted once the packaging issue is resolved :)

I though about get_psutil() function, do you think that it is better? But it will be a meaningless function once the issue is fixed.